### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by fidian**
Here's the library I've been promising.

### CODE REVIEW

**Comment by fidian** [comment]
Added a fudge factor to times that allow the tests to pass on Node 4. Looks like a `setTimeout(function, 100)` might get called only 99 ms later.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CHANGELOG
+=========
+
+
+1.0.0 - YYYY-MM-DD
```
Comment (GoogilyBoogily) [inline CHANGELOG.md]: Should probably have today's date.

File: spec/lib/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+describe("lib/index.js", () => {
+    var morePromises, timer;
+
+
+    /**
+     * Create a delayed promise that resolves or rejects
+     *
+     * @param {number} ms
+     * @param {*} result
+     * @param {boolean} [isFailure=false]
```
Comment (GoogilyBoogily) [inline spec/lib/index.spec.js:12]: Doesn't look like isFailure actually defaults to `false`. Granted, `undefined` and `false` are treated the same way, but technically, there is a difference.

**Review by GoogilyBoogily — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (fidian) [reply CHANGELOG.md]: Corrected. 60cbb5e

**Review by GoogilyBoogily — APPROVED** [review]
Looks good to me.

**Pull request merged**